src/app/views/main/main.component.scss

```diff
@@ -1,32 +1,30 @@
+@import 'global/sizes', 'global/colors';
+
 .ccf-main.wrapper {
   .container {
-    .ccf-leftbar {
-      position: fixed;
-      z-index: 2;
+    .header {
+      height: $header-height;
     }
-    .drawer-wrapper {
+
+    .contents-area {
       display: flex;
-      height: calc(100vh - 3.1875rem);
-      margin-left: 4.25rem;
-      .container {
+      height: calc(100vh - #{$header-height});
```

Comment:
Should that be a $ instead of a #?

Reply:
It is interpolation, otherwise the css ends up with calc(100vh - $header-height), which is invalid

Reply:
For some reason it can't figure it out on its own

Reply:
Huh weird. Ok.